library/Ivoz/Ast/Domain/Service/PsEndpoint/UpdateByUser.php

```diff
@@ -7,11 +7,6 @@
 use Ivoz\Provider\Domain\Model\User\UserInterface;
 use Ivoz\Provider\Domain\Service\User\UserLifecycleEventHandlerInterface;
 
-/**
- * Class UpdateByUser
- * @package Ivoz\Ast\Domain\Service\PsEndpoint
- * @lifecycle pre_persist
- */
 class UpdateByUser implements UserLifecycleEventHandlerInterface
```

Comment:
Hey! Is this incorrect? You removed the header of a Lifecycle service.

Reply:
Yes, it's on purpose. This is a post_persist service, documentation was wrong (and it's not the only one).

Honestly, I' would remove them all